packages/veritone-react-common/src/components/Video/MediaPlayer/styles.scss

```diff
@@ -0,0 +1,45 @@
+.mediaPlayer {
+  font-size: 12px;
+  font-family: Roboto;
+  outline: none;
+
+  .mediaPlayButton {
+    top: 50%;
+    left: 50%;
+    margin-left: -0.75em;
+    margin-top: -0.75em;
+    height: 1.5em;
+    width: 1.5em;
+    border-radius: 1em;
+    border: none;
+  }
+
+  .mediaPlayerControls {
+    :global(.video-react-control) {
+      width: 3em;
+    }
+
+    :global(.video-react-time-control) {
+      padding-left: 0.5em;
+      padding-right: 0.5em;
+    }
+
+    :global(.video-react-control-text) {
```

Comment:
This can be reduced to -

```
:global(.video-react-context-text),
:global(.video-react-mouse-display),
:global(.video-react-play-progress::after) {
  font-family: Roboto !important;
}
```